content/SCALE/SCALECLIReference/Account/CLIgroup.md

```diff
@@ -10,8 +10,233 @@ tags:
 
 {{< toc >}}
 
-
 {{< include file="/_includes/CLI/CLIGuideWIP.md" type="page" >}}
 
+The **group** namespace contains 8 commands and is based on functions found in the SCALE API and web UI. It provides access to group account creation, configuration, and management.
+
+You can enter commands from the main CLI prompt or from the system namespace prompt.
+
+### The create Command
+The `create` command creates a new group.
+
+{{< expand "Using the create Command" "v" >}}
+From the CLI prompt, enter:
+
+`account group create`
+
+From the **account** prompt, enter:
+
+`group create`
+
+{{< expand "Command Example" "v" >}}
+{{< include file="content/_includes/CLI/AccountGroupCreateExample.md" type="page" >}}
+{{< /expand >}}
+{{< /expand >}}
+
+`create` has one command option and 7 arguments.
+
+### The create Interactive Arguments Editor
+
+Entering the `create --` option will open an **interactive arguments editor**.
```

Comment:
never use "will" in the articles unless mentioning a future change to the feature, such as this command will be removed in a future release, suggestion for active voice present tense:
```suggestion
Entering the `create --` option opens an **interactive arguments editor**.
```